tests/test_derivatives.py

```diff
@@ -77,13 +77,17 @@ def test_fun(x, y, a):
         y = np.array([60, 1, 100, 0.02])
         a = -2
 
-        jac_AD = AutoDiffDerivative(test_fun, argnum=0)
-        J_AD = jac_AD.compute(x, y, a)
+        jacf_AD = AutoDiffDerivative(test_fun, argnum=0, mode="fwd")
+        Jf_AD = jacf_AD.compute(x, y, a)
+        jacr_AD = AutoDiffDerivative(test_fun, argnum=0, mode="rev")
+        Jr_AD = jacr_AD.compute(x, y, a)
 
-        jac_FD = AutoDiffDerivative(test_fun, argnum=0)
+        jac_FD = FiniteDiffDerivative(test_fun, argnum=0)
```

Comment:
this was wrong on master, it should be comparing FD to AD but was comparing AD to itself